hack/inventory/test-sample/rook/cluster.yaml

```diff
@@ -78,31 +68,40 @@ spec:
 #      requests:
 #        cpu: "1"
 #        memory: "1024Mi"
+# The option to automatically remove OSDs that are out and are safe to destroy.
   removeOSDsIfOutAndSafeToRemove: false
   priorityClassNames:
     all: rook-ceph-default-priority-class
+  storage:
+    useAllNodes: false
```

Comment:
Node and device information is better to be commented out, since hcsctl user should enter those information based on their environment. And for the test environment, `useAllNodes: true, useAllDevices: true` is recommended.

Reply:
Reflected thank you